devolving_music/lib/Song_Scores.py

```diff
@@ -0,0 +1,32 @@
+from devolving_music.lib.elo_scoring import *
```

Comment:
There needs to be at least one newline (ideally two) between imports and the class.

Try installing [pylint](https://code.visualstudio.com/docs/python/linting) for VSCode so it can point out stuff like this.